### DISCUSSION TITLE
[Heartwarming] - Goodbye.

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8879] [appreciation]
Just want to say good luck to you all. Once again you're all doing really great job. That was fun to contribute into it, but i guess this enthusiasm is exhusted. 
I saw how DDLC was changing from older versions and it's impressive. And i will stay with Monika, i had good times and i don't want to forget it. 
Still, i have some undone things here, i don't want to do anything with it, my mental state was unstable these months barely was adequate but i'm still alive. 
Miracle i could say, these breakdowns is tough to handle. Anyway it's my problems.
So yeah, Goodbye. Just know you're given me happiness even if that wasn't so long, that was true for me. That's all, GL.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
